.github/workflows/test.yml

```diff
@@ -4,9 +4,8 @@
 name: tox
 
 on:
-  push:
-    branches: [ master ]
   pull_request:
+  push:
```

Comment:
This enables the CI runs on pull requests not inteded to be merged into the `master` branch.